app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -129,6 +131,10 @@ watch(selectedIndex, (_index) => {
   })
   showDropdownWidget.value = false
 })
+
+const childWidgetRef = ref<HTMLElement>()
+const childSize = useResizeObserver(childWidgetRef)
+const arrowPos = computed(() => (childSize.value != null ? childSize.value.x / 2 : 0))
```

Comment:
I don't quite like that we position this arrow in a programmatic way. Perhaps we could do some CSS trick here @Frizi ?

Reply:
We could. The arrow could be centered by CSS in many ways. Here probably the esasiest would be `margin: 0 auto;`, or `left: 50%; margin-left: -50%;`

Reply:
Fixed